docs/how-to-use.md

```diff
@@ -25,6 +25,15 @@ cd background-checks
 
 This starts up a new Docker Application with a stack of containers.
 
+## How to monitor the application
+
+Once the applications has started you can navigate to [http://localhost:3000/](http://localhost:3000/)
+and log into Grafana with credentials `admin`, `admin`.
```

Comment:
I believe we can set grafana to not require any authentication, which is fine for our use case.

Reply:
ok. i didnt know how will look

Reply:
fixed, no more login screen

Reply:
Awesome, so please update the doc :)